src/pages/TabsPage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Tab } from '../types/Tab';
+import { TabsTable } from '../componets/TabsTable';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId = '' } = useParams();
+
+  return (
+    <>
```

Comment:
You have a container to wrap your content

```suggestion
    <div className="container">
```